workers/unity/Packages/io.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -330,12 +330,39 @@ public static void Clean()
                 return true;
             }
 
+            if (buildTarget == BuildTarget.StandaloneOSX)
+            {
+                // Unity executable name has changed on MacOS
+                // This is a temp work around our Launcher trying to run the wrong file.
+                // See [UTY-2294]
+                CreateLaunchJson(workerBuildData.PackageName, Application.productName);
+            }
+
             var zipPath = Path.Combine(PlayerBuildDirectory, workerBuildData.PackageName);
             var basePath = Path.Combine(Common.BuildScratchDirectory, workerBuildData.PackageName);
             Zip(zipPath, basePath, targetEnvironment == BuildEnvironment.Cloud);
             return true;
         }
 
+        private static void CreateLaunchJson(string packageName, string productName)
+        {
+            var basePath = Path.Combine(Common.BuildScratchDirectory, packageName);
+            using (var jsonWriter = File.CreateText(Path.Combine(basePath, "launcher_client_config.json")))
+            {
+                var json = @"{
+  ""command"": " + $"\"./{packageName}.app/Contents/MacOS/{productName}\"," + @"
```

Comment:
Oh my this string interpolation is hard to read

Reply:
Blame C# for not allowing both @ and $

Reply:
What, yes you can

Reply:
https://stackoverflow.com/questions/31638579/how-do-you-use-verbatim-strings-with-interpolation

Reply:
I see, that will mess with the `{}` usage in the arguments...
Do you want me to update this or leave it?

Reply:
Just leave it as is :)